### PR TITLE
[k8s][feat] Migrate from v1 config

### PR DESCRIPTION
**Pull request by aquamatthias**

### CODE REVIEW

File: resotolib/resotolib/config.py [hunk 1]
```diff
@@ -125,7 +125,10 @@ def load_config(self, reload: bool = False) -> None:
                 for config_id, config_data in config.items():
                     if config_id in Config.running_config.classes:
                         log.debug(f"Loading config section {config_id}")
-                        new_config[config_id] = jsons.load(config_data, Config.running_config.classes[config_id])
+                        clazz: type = Config.running_config.classes.get(config_id, Any)
+                        # use the from_json class from config, if available
+                        loader = getattr(clazz, "from_json", jsons.load)
+                        new_config[config_id] = loader(config_data)
```
Comment (lloesche) [inline resotolib/resotolib/config.py:131]: Doesn't jsons.load require a second argument that is the class name you want the json data to marshal to?

Comment (aquamatthias) [reply resotolib/resotolib/config.py]: absolutely right. fixed.

**Comment by lloesche** [comment]
Fails on a default 2.2 config with
```
22-06-17 22:06:28|resotoworker| INFO|56932|MainThread  Migrate k8s configuration from v1
Traceback (most recent call last):
  File "/Users/lukas/repo/resoto/venv/bin/resotoworker", line 33, in <module>
    sys.exit(load_entry_point('resotoworker', 'console_scripts', 'resotoworker')())
  File "/Users/lukas/repo/resoto/resotoworker/resotoworker/__main__.py", line 89, in main
    config.load_config()
  File "/Users/lukas/repo/resoto/resotolib/resotolib/config.py", line 131, in load_config
    new_config[config_id] = loader(config_data)
  File "/Users/lukas/repo/resoto/plugins/k8s/resoto_plugin_k8s/base.py", line 326, in from_json
    for i in range(len(config))
TypeError: object of type 'NoneType' has no len()
```

**Review by meln1k — APPROVED** [review]
LGTM

**Pull request merged**